src/xenserver/opt/xensource/www/XCP-ng-index.html

```diff
@@ -1,297 +1,24 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml">
 <!--
     InstanceBegin template="/Templates/lcm_check_template.dwt" codeOutsideHTMLIsLocked="false"
   -->
 
 <head>
-  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-  <script src="asset/jquery-3.6.0.min.js"></script>
-  <script src="asset/jquery.modal.min.js"></script>
+  <meta charset="utf-8">
+  <title>Welcome to XCP-ng @PRODUCT_VERSION@</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <script src="asset/deploy.js"></script>
   <script>
     function copyToClipboard(element) {
-      var $temp = $('<input>');
-      $('body').append($temp);
-      $temp.val($(element).text()).select();
-      document.execCommand('copy');
-      $temp.remove();
+      element = typeof element === "string" ? document.querySelector(element) : element;
+      navigator.clipboard.writeText(element.textContent)
     }
   </script>
```

Comment:
Is this used?

Reply:
That's true, that I haven't seen any code that call this function
I remove it now